docs/content/faq.md

```diff
@@ -4,16 +4,66 @@ description: Frequently Asked Questions
 ---
 
 * [What is CNAB?](#what-is-cnab)
+* [Does Porter fully implement the CNAB specification?](#does-porter-fully-implement-the-cnab-specification)
+* [Does Porter solve something that Ansible, Terraform, etc does not?](#does-porter-solve-something-that-ansible-terraform-etc-does-not)
 * [Does Porter Replace Duffle?](#does-porter-replace-duffle)
 * [Should I use Porter or Duffle?](#should-i-use-porter-or-duffle)
-* [How does your release naming scheme work?](#how-does-your-release-naming-scheme-work)
+* [If an upgrade fails, can I roll back?](#if-an-upgrade-fails-can-i-roll-back)
+* [How do I run commands that aren't in the default invocation image?](#how-do-i-run-commands-that-aren-t-in-the-default-invocation-image)
+* [Are mixins just wrappers around OS or executable calls?](#are-mixins-just-wrappers-around-os-or-executable-calls)
 
 ## What is CNAB?
 
 CNAB stands for "Cloud Native Application Bundle". When we say "bundle", that is what
 we are referring to. There is a CNAB Specification and you can learn more about
 it at [cnab.io](https://cnab.io).
 
+We like to think of bundles as "cloud installers". They handle installing not just your
+application, but also the underlying infrastructure that your application depends upon.
+
+For example, let's say that you had a Wordpress bundle, which deploys on to a
+Kubernetes cluster and relies on a MySQL as a Service database. The bundle could
+have everything you need inside of it:
+
+* The Wordpress chart
+* The installation bash script
+* Client binaries for helm and azure
+
+The installer would take care of:
+
+1. Requesting credentials for an existing Kuberentes cluster.
+1. Provisioning an Azure MySQL database.
+1. Collecting the database credentials.
+1. Installing the Wordpress chart and passing in the database credentials.
+
+The person managing the application would only need to know `porter install` and have
+credentials for a Kubernetes cluster.
+
+## Does Porter fully implement the CNAB specification?
+
+Porter currently implements much of the CNAB spec, however, as the [CNAB
+specification](https://github.com/deislabs/cnab-spec) moves toward 1.0, some
+gaps have emerged. Currently, if you build a bundle with Porter, you'll be able
+to install it with Porter. There are some gaps with the spec that limit
+compatibility with other CNAB tooling. See the [CNAB 1.0
+Milestone](https://github.com/deislabs/porter/milestone/12) for more information
+on these gaps.
+
+## Does Porter solve something that Ansible, Terraform, etc does not?
+
+Porter is an implementation of the CNAB specification. Cloud Native Application
+Bundles is a different way of answering "_How do I reliably, securely deploy an
+application and its infrastructure?_". It isn't replacing ansible or terraform
```

Comment:
I wonder if we should capitalize `Ansible`, `Terraform` and perhaps other referenced tools (aside from cases when it is referencing a cli, e.g., `kubectl`, `terraform`, etc.)